tools/webpack.config.babel.js

```diff
@@ -95,6 +108,11 @@ const createConfig = (env = {}) => {
     },
     // What information should be printed to the console
     stats: 'minimal',
+    performance: {
+      hints: isProductionBuild ? 'error' : false,
```

Comment:
@klinzo do you think this is dangerous? 

This line causes release builds to fail. If we do not build release builds on "commit" then we firstly know about big bundle sizes when merging to develop.

I think it is better to always 'error'.

Reply:
Else it is maybe not clear why some builds fail and some not.

Reply:
Yeah builds may fail, which is not dangerous per se, but rather good imo. If the bundle sice becomes a lot bigger than it is right now, it is better to have a failing build than to release it without checking.

Reply:
We do production builds on commit now, too. This should not be a problem therefore.

Reply:
We shouldn't error for dev builds as they are **way** larger (5MB or so) since all minimifcation etc. are turned off. Hence it only makes sense to check this for production builds.